Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -262,8 +262,14 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual(project.creator.id, dataLakeClientProperties?["project_creator_uid"] as? Int)
     XCTAssertEqual(24 * 15, dataLakeClientProperties?["project_hours_remaining"] as? Int)
     XCTAssertEqual(30, dataLakeClientProperties?["project_duration"] as? Int)
-    XCTAssertEqual(1_476_657_315, dataLakeClientProperties?["project_deadline"] as? Double)
-    XCTAssertEqual(1_474_065_315, dataLakeClientProperties?["project_launched_at"] as? Double)
+    XCTAssertEqual(
+      1_476_657_315.toISO8601DateTimeString(),
```

Comment:
Instead of calling the function, I think the assertion should just be the literal of the expected value.

Reply:
While I agree with you that this can be the literal value. `1_476_657_315.toISO8601DateTimeString()` , because I considered it as well. I'm of the opinion that, the true test of `toISO8601DateTimeString()` is to do the conversion this way instead of literal value. The test is undergoing the same definition for the property as it's done in KSRAnalytics. 

This is my opinion, let me know what you think.

Reply:
What you're saying definitely makes sense. My concern is mostly a readability one. This test shouldn't be verifying if the extension works (you wrote some specific tests that do that) but instead should have a clear, readable expectation in the form of the output string.